Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.4.1' if ENV['DYNO']
```

Comment:
is that conditional needed? last time i checked it caused issues with rvm, as it uses a crappy regex to extract the ruby version. so i think we generally just put `ruby '2.4.1'` and bundler supports it fine.